### ISSUE TITLE
Windows 10

### ISSUE THREAD

**Issue by AlexChea** [Microsoft/ILMerge#15]
hi,
the latest dot net 4.7 is not support

**Comment by mike-barnett** [comment]
Could you please be more specific? How is it not supported? If you can provide a repro, that would be best. Thanks!

**Comment by Fabi** [comment]
.net 4.7 exe/dll merge works fine for me!

**Comment by RicardoASilva** [comment]
I might have the same issue, tried out the ILMerge GUI tool and I'm getting an error that "Cannot merge .Net 4.7 with ILMerge." although could be related to the tool and not ILMerge itself.

**Comment by Fabi** [comment]
It's related to the GUI I think, because I had the same problems there, but it works fine with the command line version

**Comment by AlexChea** [comment]
I see, i am using GUI to merge too, will try using command line.
thanks @Fabi

**Comment by michaelfp** [comment]
Please just update your minor version of framework to 7.
For who would like to use ilmergeGUI. Get from this repo https://github.com/jpdillingham/ILMergeGUI, this guy made the fix and is worked. 

It's a simple fix,

**Comment by AlexChea** [comment]
is it possible to use ilmerge in Windows 10 to merge into dot net 4.6?

**Comment by mike-barnett** [comment]
Windows 10 should have nothing to do with it: the only thing that should matter is the version of .NET that you have. I am not aware of any problems with 4.6. If you have a (small) repro of any problem, please send it to me.

**Comment by michaelfp** [comment]
@AlexChea, yes it´s possible! If you have only 4.6 version installed worke as well. But if you have already installed 4.7 need to get from the repo above.

**Comment by mike-barnett** [comment]
Since I have nothing to do with the GUI, it would help me a lot if people would use this site for issues having to do with ILMerge itself. Thanks!

**Comment by michaelfp** [comment]
@mike-barnett correct. I´m just trying to help. With run the ilmerge.exe in command-line work´s great in Windows 10.

**Comment by mike-barnett** [comment]
Sorry! I didn't mean that to sound nasty. I'm super appreciative of your help. I just meant that I can be more efficient if I know whether an issue is about ILMerge itself or the GUI.

**Comment by AlexChea** [comment]
@mike-barnett in Windows 10, merge gui did not provide option for dotnet 4.6.x, only dotnet 4
how to make it available?

**Comment by mike-barnett** [comment]
I'm sorry, but I really do not know anything about the GUI. Those questions should be addressed to its author. If you're using the command line (or in-memory API), then you would use the /targetplatform option to tell ILMerge which .NET version to use.

**Comment by weedkiller** [comment]
On windows 10, I have the same error with the latest version

![image](https://user-images.githubusercontent.com/21086190/32030007-052186e4-b9c6-11e7-9377-f729474f97f3.png)

**Comment by mike-barnett** [comment]
That seems like a problem with the GUI, not ILMerge. Please report it to the author of that tool. Thanks!

**Comment by jnm2** [comment]
@weedkiller You're in the wrong issues site. See https://bitbucket.org/wvd-vegt/ilmergegui/issues/1/net-47-framework-support

**Comment by 3Tecnos-Development** [comment]
To solve this problem simply go in source code at line 1192 and add the condition:
![ilmergegui](https://user-images.githubusercontent.com/26663404/35567346-e8fe4236-05ab-11e8-816f-87ac7a747944.gif)

Whoever does not want to have this job, I've compiled a version fixing the bug:
[ILMergeGui.zip](https://github.com/Microsoft/ILMerge/files/1677725/ILMergeGui.zip)
